Containers/talk-recording/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM python:3.11.3-alpine3.18
+
+COPY --chmod=775 start.sh /start.sh
+COPY --chmod=664 recording.conf /etc/recording.conf
+
+RUN set -ex; \
+    apk add --no-cache \
+        ca-certificates \
+        tzdata \
+        bash \
+        ffmpeg \
+        libpulse \
+        bind-tools \
+        netcat-openbsd \
+        git \
+        wget \
+        shadow; \
+    # xvfb firefox chromium chromium-chromedriver?
+    # apk add --no-cache geckodriver --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing; \
```

Comment:
```suggestion
    shadow \
    xvfb \
    firefox \
    openssl; \
    # chromium chromium-chromedriver?
    # apk add --no-cache geckodriver --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing; \
```

Reply:
can you say me where thoose deps are listed?

Reply:
you mean firefox?

Reply:
yes, where docs that list thoose deps

Reply:
openssl is needed for chpasswd, right?

Reply:
yes

Reply:
Firefox and others are listed in https://github.com/nextcloud/spreed/pull/8788/files

Reply:
or we download the binary file from theier github releases

Reply:
> so we need to pull geckodriver from alpine edge/testing again?

I guess so...

Reply:
done

Reply:
should we request again to move it out of testing?

Reply:
Indeed, maybe we could ask the people at alpine aports again to move it out of testing and then also create a follow-up ticket here?